controllers/toplevel/legacyPages.js

```diff
@@ -0,0 +1,97 @@
+const config = require('config');
+const moment = require('moment');
+const ab = require('express-ab');
+const { proxyLegacyPage, postToLegacyForm } = require('../../modules/proxy');
+const { legacyProxiedRoutes } = require('../routes');
+const cached = require('../../middleware/cached');
+
+function initAwardsForAll(router) {
+    function proxyWithoutChanges(req, res) {
+        return proxyLegacyPage(req, res, dom => dom);
+    }
+
+    function proxyAwardsForAllTemporaryChangeMe(applyUrl, req, res) {
+        return proxyLegacyPage(req, res, dom => {
+            res.set('X-BLF-Legacy-Modified', true);
+
+            const applyTab = dom.window.document.querySelector('#mainContentContainer .panel:last-of-type');
+
+            const additionalText = `
+                <h4>Apply online</h4>
+                <p><br/><a class="roundedButton blueButton" style="display: inline-block;" href="${
+                    applyUrl
+                }">Apply Online</a></p>
+                <h4>Apply by post</h4>
+            `;
+
+            applyTab.innerHTML = additionalText + applyTab.innerHTML;
```

Comment:
Is it worth doing a sanity check here to verify we made a change or found the right DOM elements and reporting to Sentry if not? Just to catch the edge case where these pages get modified...

Reply:
Good shout